lib/fog/google/models/storage_json/directories.rb

```diff
@@ -0,0 +1,37 @@
+require "fog/core/collection"
+require "fog/google/models/storage_json/directory"
+
+module Fog
+  module Storage
+    class GoogleJSON
+      class Directories < Fog::Collection
+        model Fog::Storage::GoogleJSON::Directory
+
+        def all
+          data = service.get_service.body["items"]
+          load(data)
+        end
+
+        def get(key, options = {})
+          remap_attributes(options,             :delimiter  => "delimiter",
+                                                :marker     => "marker",
+                                                :max_keys   => "max-keys",
+                                                :prefix     => "prefix")
+          data = service.get_bucket(key, options).body
+          directory = new(:key => data["name"])
+          # options = {}
```

Comment:
It might be related to how a `directory` object gets its associated `files`, but I'm not really sure. That might be handled by `list_objects`. In which case, we should ditch it.

So far things seem to be working without it. So maybe let's ditch it and revisit if needed.